From 2abf1fc91d8139fa4f8b19b06f649af191224242 Mon Sep 17 00:00:00 2001
From: Mike Snitzer <snitzer@redhat.com>
Date: Sat, 9 Dec 2017 20:38:16 -0500
Subject: [PATCH] dm: remove stale comment blocks
Git-commit: 2abf1fc91d8139fa4f8b19b06f649af191224242
Patch-mainline: v4.16-rc1
References: bsc#1093023

These CRUD comments have worn out their welcome.  The code is what it
is, over time it'll hopefully get better.  But these comments serve no
purpose whatsoever.

Signed-off-by: Mike Snitzer <snitzer@redhat.com>
Signed-off-by: Coly Li <colyli@suse.de>

---
 drivers/md/dm.c | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/drivers/md/dm.c b/drivers/md/dm.c
index 308d178fff73..5827e1641ba2 100644
--- a/drivers/md/dm.c
+++ b/drivers/md/dm.c
@@ -752,15 +752,6 @@ int dm_set_geometry(struct mapped_device *md, struct hd_geometry *geo)
 	return 0;
 }
 
-/*-----------------------------------------------------------------
- * CRUD START:
- *   A more elegant soln is in the works that uses the queue
- *   merge fn, unfortunately there are a couple of changes to
- *   the block layer that I want to make for this.  So in the
- *   interests of getting something for people to use I give
- *   you this clearly demarcated crap.
- *---------------------------------------------------------------*/
-
 static int __noflush_suspending(struct mapped_device *md)
 {
 	return test_bit(DMF_NOFLUSH_SUSPENDING, &md->flags);
@@ -1497,9 +1488,6 @@ static void __split_and_process_bio(struct mapped_device *md,
 	/* drop the extra reference count */
 	dec_pending(ci.io, errno_to_blk_status(error));
 }
-/*-----------------------------------------------------------------
- * CRUD END
- *---------------------------------------------------------------*/
 
 /*
  * The request function that remaps the bio to one target and
-- 
2.16.3

